django/contrib/contenttypes/fields.py

```diff
@@ -389,6 +389,20 @@ def resolve_related_fields(self):
             )
         ]
 
+    def get_local_related_value(self, instance):
+        return self.get_instance_value_for_fields(instance, self.foreign_related_fields)
+
+    def get_foreign_related_value(self, instance):
```

Comment:
I can remove this without a test failure. Could you write a test so I understand why this is required?

Reply:
@sarahboyce just adjusted the test.

The reason why it was previously passing without adjustments on SQLite is that the `Bookmark` and `TaggedItem` auto-incrementing primary keys were _aligned_ so `TaggedItem.id` and `.object_id` had the same value so it didn't matter if the wrong field was picked up for building the the associative map

```python
{'_prefetched_objects_cache': {},
 '_state': <django.db.models.base.ModelState object at 0x1057df5c0>,
 'content_type_id': 22,
 'created_by_ct_id': None,
 'created_by_fkey': None,
 'favorite_ct_id': 22,
 'favorite_fkey': '1',
 'id': 1,
 'object_id': 1,
 'tag': ''}
```

I created a second `Bookmark` in the test to ensure they are offset adequately.